flasher_stub/ld/rom_32c61.ld

```diff
@@ -0,0 +1,411 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */

```

Comment:
Please remove the license. It is not in any of the other ld files.

Reply:
Thank you @radimkarnis for the review!